src/basic/fs-util.c

```diff
@@ -33,9 +33,9 @@
 #include "umask-util.h"
 #include "user-util.h"
 
-int unlink_noerrno(const char *path) {
+int unlinkat_noerrno(int dir_fd, const char *path, int flags) {
```

Comment:
honestly, this call seems a bit usless these days where we copy out the error from errno immediately always. it's a left-over from a different era where our error handling was still more all over the place